tests/integration/cqlengine/test_ttl.py

```diff
@@ -151,6 +156,7 @@ def test_ttl_included_with_blind_update(self):
         self.assertIn("USING TTL", query)
 
 
+@unittest.skipIf(CASSANDRA_VERSION < '2.0', "default_time_to_Live was introduce in C* 2.0, currently running {0}".format(CASSANDRA_VERSION))
```

Comment:
Did you come to an understanding of how this works between unittest/unitest2 and runtimes?